src/database/uiTestResult.ts

```diff
@@ -0,0 +1,56 @@
+/** @ignore */
+/*
+ * Copyright (c) 2025 Certinia Inc. All rights reserved.
+ */
+import { UiTestResult } from './entity/uiTestResult';
+import { saveRecords } from './saveRecords';
+import { getConnection } from './connection';
+
+/**
+ * Data Transfer Object for UI Test Results
+ */
+export interface UiTestResultDTO {
+  id: number;
```

Comment:
ID is the autogenerated number primary key. If you insert values with it already set, then the save process will upsert the records (replace existing). Should remove from this interface, since we either already have the data or we're using `WHERE` on unique field combos to locate different records.